chia/data_layer/data_layer_errors.py

```diff
@@ -0,0 +1,40 @@
+from typing import Iterable, List
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+
+
+class IntegrityError(Exception):
```

Comment:
Better log this. Not connected yet?

Reply:
I'm not quite sure what you mean.  This is a base class to group some of the other exceptions below in the exception hierarchy.

Reply:
Ah, I see